packages/spacebars-compiler/package.js

```diff
@@ -5,6 +5,11 @@ Package.describe({
   git: 'https://github.com/meteor/blaze.git'
 });
 
+// Use uglify-js from NPM
```

Comment:
I mean, this comment is unnecessary. It is pretty clear that this is what this code does.